torch/nn/quantized/functional.py

```diff
@@ -478,9 +499,9 @@ def elu(input, alpha=1., inplace=False, scale=None, zero_point=None):
         torch._C._nn.elu(input, alpha, out=output)
         return output
     elif inplace:
-        return torch._C._nn.elu_(input, alpha)
+        return torch._C._nn.elu_(input, alpha, 1., 1.)
```

Comment:
these have a default of 1 in `native_functions.yaml`, would it work to revert this and next line?